packages/fast-components-styles-msft/src/utilities/color/recipe.ts

```diff
@@ -0,0 +1,113 @@
+import {
```

Comment:
Note on the file name - everything about these color utilities relates to recipes - can we  just put this stuff in common? Or maybe accessability.ts?

Reply:
Yeah, funny name. I thought I had something better. I was trying to encapsulate it and avoid cluttering common. I'll think of something more descriptive.